src/settings/OrderTemplates/OrderTemplatesEditor/OrderTemplatesEditor.js

```diff
@@ -19,7 +19,9 @@ import {
   PaneMenu,
   Row,
 } from '@folio/stripes/components';
+import { EditCustomFieldsRecord } from '@folio/stripes/smart-components';
 import stripesForm from '@folio/stripes/final-form';
+import { Field } from 'react-final-form';
```

Comment:
fix the order, please. General rule: Import orders should be in hierarchical and alphabetic
Ex: 
```
import constants from '../../constants';
import anotherConstants from '../constants'
import somePackage from '../utils'
```
external packages
--- leave an empty line ---
folio packages
--- leave an empty line ---
internal packages